src/sorcha_cmdline/main.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+import subprocess
+import sys
+import shutil
+import os
+
+#
+# Generic verb dispatcher code
+#
+
+def find_sorcha_verbs():
+    """Find available sorcha commands in the system's PATH."""
+    sorcha_verbs = []
+    for directory in os.environ.get("PATH", "").split(os.pathsep):
+        if os.path.isdir(directory):
+            for item in os.listdir(directory):
+                if item.startswith("sorcha-") and os.access(os.path.join(directory, item), os.X_OK):
+                    sorcha_verbs.append(item[len("sorcha-"):])
+    return sorted(set(sorcha_verbs))
+
+def main():
+    # Discover available sorcha verbs
+    available_verbs = find_sorcha_verbs()
+
+    if not available_verbs:
+        print("Error: No available 'sorcha-' utilities found.")
+        sys.exit(1)
+
+    # Set up the argument parser with epilog text
+    description = 'Sorcha survey simulator suite.'
+    epilog_text = "These the most common sorcha verbs:\n\n" \
+                  "   run       Run a simulation\n" \
+                  "   demo      Run a demo\n" \
```

Comment:
I understand that these descriptions are written to be general, in case we want to add more commands later. However, I think "run a demo" here is misleading. It makes it sound like the command runs the demo in the same way `sorcha run` would run the simulation. Maybe "Set up a demo simulation"?

Reply:
Fixed.